docusaurus/docs/gers/gers.mdx

```diff
@@ -33,44 +31,39 @@ D SELECT id FROM read_parquet('s3://overturemaps-us-west-2/release/2024-01-17-al
 │ 08ba00858a164fff020008cf146d28e6 │
 │ 08ba00858a16cfff0200271c60f0a5ea │
 │ 08ba00858a161fff02008633c8f27bf7 │
-│ 08ba00858a163fff0200bf648279e290 │
-│ 08ba00858a163fff02001756776dd6ca │
-│ 08ba00858a163fff0200e04e60516f5c │
-│ 08ba00858a16efff02008127dee56493 │
-│ 08ba00858a16efff020065a1f1e26a85 │
-│ 08ba00858a144fff020057e23ab30103 │
+│ 08ba00858a163fff0200bf648279e290 │ │
 ├──────────────────────────────────┤
-│             10 rows              │
+│             5 rows               │
 └──────────────────────────────────┘
 ```
 
-## Stability and traceability
+Overture discourages users from attempting to glean information about a feature from its GERS ID. The ID should be treated as a unique identifier and nothing more. It is worth going into more detail about Overture's use of the [hexagonal hierarchical geospatial indexing system (H3)](https://h3geo.org/) in GERS. H3 is used primarily to prevent hashing conflicts among features with similar attributes. The H3 index component of a GERS ID reflects a feature's location in the H3 grid system when Overture first assigns it an ID. Because GERS IDs are intended to be stable over time, Overture will not generate a new ID for a feature if its moves within the H3 cell it was originally assigned.  
 
-GERS IDs are stable (with a reasonable tolerance of error) across multiple versions of Overture data. We strive to ensure the mapping of a real-world thing to a GERS ID remains consistent. When stability is not possible, traceability will be provided. For example:
 
-- a single road segment is bisected by a new road and becomes two road segments: 1 GERS ID → 2 new GERS IDs
-- one large building footprint on the map is determined to be four smaller buildings when a higher resolution dataset becomes available: 1 GERS ID → 4 new GERS IDs
-- a building is shifted 10m west when higher resolution imagery is made available: GERS ID is preserved for that feature
 
-## Getting started
+## Stability and traceability
+
+GERS IDs are stable (within a reasonable tolerance of error) across multiple versions of Overture data. When stability is not possible, traceability will be provided. For example:
```

Comment:
i would clarify we don't have this yet. or remove it altogether for now till we have at least the design/schema for modelling traceability.